### PR TITLE
[READY]Multi Z interaction & construction & tweaks

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request

Some MultiZ tweaks

Implement flags that block move up/down, and from up/down. Add block from down to catwalk, so you no more can climb up on catwalk through catwalk.

Maybe implement it to block air, some later.

## Why It's Good For The Game
MultiZ is great

## Changelog
:cl:
tweaks: MultiZ
tweaks: On openspace edge you can look up/down
tweaks: No more climbing through catwalk up.
tweaks: Spacemens studied  to carefully crawl on lattice.
tweaks: Now spacemens can interract with things above/below it.
add: Look down verb
/:cl:

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Whats with the commented-out stuff?

**Comment by Dennok** [comment]
> Whats with the commented-out stuff?

First i think make full check on base of turf, but while open turf has floor it unnecessary.

File: code/game/turfs/open/openspace.dm [hunk 1]
```diff
@@ -49,15 +49,37 @@ GLOBAL_DATUM_INIT(openspace_backdrop_one_for_all, /atom/movable/openspace_backdr
...
+				return FALSE
+		return TRUE
+	return FALSE
 
 /turf/open/transparent/openspace/zPassOut(atom/movable/A, direction, turf/destination)
 	if(A.anchored)
 		return FALSE
-	for(var/obj/O in contents)
-		if(O.obj_flags & BLOCK_Z_FALL)
+/*	for(var/obj/O in contents)
```
Comment (bobbahbrown) [inline code/game/turfs/open/openspace.dm:67]: Do not leave commented out code in production code

Comment (Dennok) [reply code/game/turfs/open/openspace.dm:67]: ups

**Comment by Koshenko** [comment]
Nice work.  Glad to see look_up is actually getting some use.  Could you possibly make it so look down also works if you're next to an openspace turf?  Perhaps check the direction the player model is looking then set perspective to a turf in same direction (not sure how to handle diagonals).

**Comment by Dennok** [comment]
Yes, i think about this.

**Comment by Dennok** [comment]
I think, it ready for now.

**Comment by Rohesie** [comment]
Do you plan to re-open this in the future?
Was this ready and working?

**Comment by Dennok** [comment]
I plan, i think it ready.
I separate it some to multiple PRs

**Pull request closed without merging**